plugins/principles-disciple/skills/pd-disable/SKILL.md

```
@@ -0,0 +1,31 @@
+
+   bash / zsh:
+   ```bash
+   node "$(ls -1d "$HOME/.codex/plugins/cache/"*/principles-disciple/*/scripts/pd-disable.cjs 2>/dev/null | sort | tail -1)"
+   ```
+
+   PowerShell:
+   ```powershell
+   node (Get-ChildItem "$env:USERPROFILE\.codex\plugins\cache\*\principles-disciple\*\scripts\pd-disable.cjs" | Sort-Object FullName | Select-Object -Last 1).FullName
+   ```
+
+   Re-enable: append ` --enable`.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)